website/integrations/services/openproject/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Integrate with OpenProject
+sidebar_label: OpenProject
+support_level: community
+---
+
+## What is OpenProject
+
+> OpenProject is a web-based project management software. Use OpenProject to manage your projects, tasks and goals. Collaborate via work packages and link them to your pull requests on Github.
+>
+> -- https://www.openproject.org/
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `openproject.company` is the FQDN of the OpenProject installation.
+- `authentik.company` is the FQDN of the authentik installation.
+
+:::note
+This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
+:::
+
+## authentik configuration
+
+To support the integration of OpenProject with authentik, you need to create a property mapping and an application/provider pair in authentik.
+
+### Create a scope mapping
+
+OpenProject requires a first and last name for each user. By default authentik only provides a full name. Therefore you need to create a property mapping to provide first and last names to OpenProject.
```

Comment:
```suggestion
OpenProject requires a first and last name for each user. By default authentik only provides a full name, as a single string value. Therefore you need to create a property mapping to provide first and last names to OpenProject.
```

Reply:
@dewi-tik is my edit technically correct? I just wanted a little more clarity about authentik only has a single value, whereas OpenProject requires two separate values...